src/TodoApp/TodoApp.test.jsx

```diff
@@ -0,0 +1,107 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TodoApp } from './TodoApp';
+
+describe('Test on <TodoApp />', () => {
+	const getElements = (screen) => {
+		const input = screen.getByLabelText('input-task');
+		const button = screen.getByLabelText('add-task');
+		const reset = screen.getByLabelText('reset-todo-list');
+		const remaining = screen.getByLabelText('remaining-tasks');
+
+		fireEvent.click(reset);
+
+		return {
+			input,
+			button,
+			reset,
+			remaining,
+		};
+	};
+
+	test('should can add 2 tasks with diferent value', () => {
```

Comment:
Test names are valuable because they serve as documentation for your code. When I don't understand what some piece of code is doing, tests often help me understand the intention.